src/test/java/org/traccar/protocol/FifotrackProtocolDecoderTest.java

```diff
@@ -25,9 +34,35 @@ public void testDecode() throws Exception {
                 "$$125,869467049296388,548,A01,38,190619025856,A,22.333905,113.590261,0,12,60,16666,0,0000,00,0,460|0|2694|13F8,1A2|4C1|0|0,B4A067,*7A"),
                 Position.KEY_DRIVER_UNIQUE_ID, "11837543");
 
+        byte[] expectedImage = new byte[13152];
+        new Random().nextBytes(expectedImage);
+        int imageSize = expectedImage.length;
+        int chunkSize = 1024;
+
         verifyNull(decoder, buffer(
                 "$$79,868345037864709,382,D05,190220085833,22.643210,114.018176,1,1,1,13152,23FFD339*25"));
 
+        for (int i = 0; i < imageSize; i += chunkSize) {
```

Comment:
Can we just test that we get the image attribute? We also probably don't need such a big buffer. We can maybe have 2-3 and then hardcode everything.

Reply:
And also we should probably separate it into its own test case.

Reply:
Something like this?

```
verifyNull(... "photo start")
verifyNull(... "large hardcoded string")
Position position = decoder.decode(... "final string")
assertNotNull(position)
assertNotNull(position.getAttributes().get(Position.KEY_IMAGE)
```

Reply:
Something like that. It would be ideal if we can do
```
verifyNull(... "photo start")
verifyNull(... "large hardcoded string")
verifyAttribute(... "final string", ...)
```